BuildResidentialHPXML/measure.rb

```diff
@@ -3529,7 +3529,7 @@ def self.set_roofs(hpxml, runner, model, args)
       hpxml.roofs.add(id: valid_attr(surface.name),
                       interior_adjacent_to: get_adjacent_to(surface),
                       azimuth: azimuth,
-                      area: UnitConversions.convert(surface.grossArea, 'm^2', 'ft^2').round(1),
+                      area: UnitConversions.convert(surface.grossArea, 'm^2', 'ft^2').round(2),
```

Comment:
Adds a second decimal to all areas so that the window areas perfectly match in the new test file.